libs/common-services/src/main/java/uk/gov/di/ipv/core/library/tracing/TracingHttpClient.java

```diff
@@ -110,6 +118,10 @@ private <T> HttpResponse<T> sendWithTracing(
     public <T> HttpResponse<T> send(
             HttpRequest request, HttpResponse.BodyHandler<T> responseBodyHandler)
             throws IOException, InterruptedException {
+        if (isClientTooOld()) {
+            LOGGER.info("Recreating HTTP client due to age");
+            return newHttpClient().send(request, responseBodyHandler);
```

Comment:
This isn't quite right - it's creating a new `TracingHttpClient` and then only using it for this single request.

Instead, I think we probably want to recreate the `baseClient` (and reassign it). It may be worth pulling the logic out into a `synchronized` method, to avoid the risk of race conditions, but it's not critical.